plugins/leemons-plugin-academic-calendar/frontend/src/hooks/keys/listRegionalConfigs.js

```diff
@@ -1,7 +1,7 @@
 export const allListRegionalConfigsKeys = [
   {
     plugin: 'plugin.academic-calendar',
-    scope: 'list-regional-configs',
+    scope: 'list',
```

Comment:
```suggestion
    action: 'list',
    scope: 'regional-config'
```

Reply:
Well, I'm technically wrong, in the all the action should not exists, but in the getList